control-plane/pkg/apis/messaging/v1beta1/kafka_channel_lifecycle.go

```diff
@@ -17,30 +17,72 @@
 package v1beta1
 
 import (
+	"sync"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
 	"knative.dev/pkg/apis"
 	duckv1 "knative.dev/pkg/apis/duck/v1"
 )
 
-// ConditionAddressable is a reference to the Addressable condition in the base package.
-// KafkaChannel uses the common Condition set defined in the base package.
+var kc = apis.NewLivingConditionSet(
+	KafkaChannelConditionTopicReady,
+	KafkaChannelConditionDispatcherReady,
+	KafkaChannelConditionServiceReady,
+	KafkaChannelConditionEndpointsReady,
+	KafkaChannelConditionAddressable,
+	KafkaChannelConditionChannelServiceReady,
+	KafkaChannelConditionConfigReady)
```

Comment:
`KafkaChannelConditionDispatcherReady` --> we won't need readiness condition for the receiver part. It is very well possible that there are no subscribers.

Reply:
Well, ok, now I remember. `KafkaChannelConditionEndpointsReady` is pretty much the same with that.

Reply:
Would it make sense to rename it, to have a better distinction between "receiver" and "dispatcher" ? To me  `KafkaChannelConditionEndpointsReady` sounds more generic, less specific

Reply:
overall, the naming of these are not good. I will now try to find better names for all

Reply:
The conditions we now have are solid. Won't touch any of these.